README.md

```diff
@@ -245,6 +245,12 @@ To request automatic tracing support for a module not on this list, please [file
 
 ## Upgrade guidelines
 
+### 0.17.0 to 0.18.0
+
+[PR-1975](https://github.com/open-telemetry/opentelemetry-js/pull/1975)
+
+- Breaking change - Update resource key collision precedence
```

Comment:
Nit: It is hard to understand without going to the source. Do you think it could be written the way that user don't need to go to spec?, for example:

```suggestion
- Breaking change - The resulting resource MUST have all attributes that are on any of the two input resources. If a key exists on both the old and updating resource, the value of the updating resource MUST be picked - previously it was opposite.
```

Reply:
Sound good to me.